universal-application-tool-0.0.1/app/models/Account.java

```diff
@@ -71,14 +72,31 @@ public void setManagedByGroup(TrustedIntermediaryGroup group) {
   }
 
   public ImmutableList<String> getAdministeredProgramNames() {
+    if (this.adminOf == null) {
+      return ImmutableList.of();
+    }
     return ImmutableList.copyOf(this.adminOf);
   }
 
+  public void setGlobalAdmin(boolean isGlobalAdmin) {
+    this.globalAdmin = isGlobalAdmin;
+    if (this.globalAdmin) {
```

Comment:
suggestion (blocking): Please document why this is necessary (I think since the CiviForm admin cannot be a program admin?)

Reply:
Done.